uaa/src/test/java/org/cloudfoundry/identity/uaa/integration/OrchestratorZoneControllerIntegrationTests.java

```diff
@@ -250,8 +271,11 @@ public void testCreateZone_ZoneAlreadyExists() {
         ResponseEntity<String> getResponseAlreadyExist = client.exchange(
```

Comment:
i think this is duplicate api call for line no 267 to 269 with response assigned getting assigned to variable getResponseAlreadyExist. Could you please delete lines 267 to 269 or arrange this to have only single call?

Reply:
In testCreateZone_ZoneAlreadyExists test case, at the beginning, a zone is being created and again there is a post call with the same name for zone already exists check.